orchagent/intfsorch.cpp

```diff
@@ -1357,6 +1381,39 @@ void IntfsOrch::doTask(SelectableTimer &timer)
     }
 }
 
+void IntfsOrch::processInbandVlanReady()
+{
+     SWSS_LOG_ENTER();
+
+     Port inbandVlan;
+     if (!gPortsOrch->getInbandPort(inbandVlan))
+     {
+         return;
+     }
+
+     string value;
+     const auto id = sai_serialize_object_id(inbandVlan.m_rif_id);
+     if (m_vidToRidTable->hget("", id, value))
+     {
+         stringstream cmd;
+         cmd << "ip link set " << inbandVlan.m_alias << " up";
```

Comment:
We cannot do "ip link" command from orchagent. This approach has to be revisited.

Reply:
@prsunny can you elaborate why we cannot do this in orchagent?

Reply:
There is no kernel interactions from orchagent. In the current architecture, this is handled by the manager daemons.

Reply:
@prsunny Thanks for the explanation. The kernel interaction was moved to intfmgr as suggested.